src/tools/rest-frontend/resources/assets/js/controllers/main/MainController.js

```diff
@@ -12,7 +12,7 @@ module.exports = function ($rootScope, $scope, Logger, $state, $anchorScroll, we
 
     // build date
     $scope.builddate = {
-        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toLocaleString()
+        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toUTCString()
```

Comment:
You can simply use toLocaleString() but also output getTimezoneOffset()/60 in parenthesis at the end. (like `Last update: Fri, 16 Dec 2016 23:14:49  (+3h UTC)`)